src/components/CartItem.tsx

```diff
@@ -1,41 +1,49 @@
-import { useState } from 'react';
+import React from 'react';
+import { useDispatch } from 'react-redux';
 import classNames from 'classnames';
 import { FiMinus, FiPlus, FiX } from 'react-icons/fi';
 
-const CartItem = () => {
-  const [itemCount, setItemCount] = useState(1);
+import { removeFromCart, increaseItemCount, decreaseItemCount } from '../redux';
+import { ICartItem } from '../types/CartItem';
+
+type Props = {
+  item: ICartItem;
+};
+
+const CartItem: React.FC<Props> = ({ item }) => {
+  const { name: itemName, price, image, count, id } = item;
```

Comment:
rename it ti product (optional)

Reply:
we already have interface IProduct, so I'll leave it as it is

Reply:
it's a prop) u can name it as u wish

Reply:
ok, I thought about ICartItem))